src/app/services/auth.service.ts

```diff
@@ -0,0 +1,22 @@
+import { Injectable } from '@angular/core';
+import { User } from '../models/user';
+import { Observable, of } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AuthService {
+  loggedIn: boolean = false;
```

Comment:
This `loggedIn` flag shouldn't be here. All of your state should be in ngrx, the service should be stateless.